tests/acceptance/dashboard-test.ts

```diff
@@ -12,8 +21,16 @@ module('Acceptance | dashboard', hooks => {
         // A fully loaded dashboard should have no major troubles
         const currentUser = server.create('user');
         const nodes = server.createList('node', 10, {}, 'withContributors');
-        server.create('node', { id: 'z3sg2', linkedNodeIds: ['1', '2', '3', '4', '5'], title: 'NNW' });
-        server.create('node', { id: '57tnq', linkedNodeIds: ['6', '7', '8', '9', '10'], title: 'Popular' });
+        server.create('node', {
+            id: noteworthyNode,
+            linkedNodeIds: ['node0', 'node1', 'node2', 'node3', 'node4'],
```

Comment:
Why not instead do?:
```
linkedNodes: nodes.slice(0, 5);
```

Reply:
Look, I'm just copying what _someone_ did before. ;) Fair enough.